pydm/utilities/generate_stylesheet_file.py

```diff
@@ -0,0 +1,166 @@
+# Utility to generate a stylesheet with all the combinations for alarms
```

Comment:
This is useful for developers, but probably shouldn't be bundled with PyDM.

Reply:
I've moved the stylesheet generating code to a different project.